Resolved  Anc visit, Multiple  dialog issues

hatanvir opened the pull request:
source: JIRA
cause: Anc visit two times given but showing one, Multiple GPS not found dialog showing
solution: Resolved this issues, getting count from visits table,

hatanvir committed:
taskId: HNPP-906
source: JIRA
cause: Anc visit two times given but showing one, Multiple GPS not found dialog showing
solution: Resolved this issues, and getting count from visits table,

hatanvir committed:
taskId: HNPP-906
source: JIRA
cause: New feature
solution: Visit date update button and required query added

mahmud6390 commented on opensrp-brac-hnpp/src/main/java/org/smartregister/brac/hnpp/utils/HnppJsonFormUtils.java:
```diff
@@ -1020,9 +1020,10 @@ public static void addLastAnc(JSONObject jsonForm,String baseEntityId,boolean is
             stepOne = jsonForm.getJSONObject(org.smartregister.family.util.JsonFormUtils.STEP1);
             JSONArray jsonArray = stepOne.getJSONArray(org.smartregister.family.util.JsonFormUtils.FIELDS);
-            String prevalue = FamilyLibrary.getInstance().context().allSharedPreferences().getPreference(baseEntityId+"_BRAC_ANC");
+            //String prevalue = FamilyLibrary.getInstance().context().allSharedPreferences().getPreference(baseEntityId+"_BRAC_ANC");
```
remove this share preference save logic. may be at formparser class baseEntityId+"_BRAC_ANC"

hatanvir committed:
taskId: N/A
source: N/A
cause: Improvement
solution: Anc data save to SharedPreferences logic removed
